packages/experimental/NativeButton/ios/MSFUIButtonViewManager.m

```diff
@@ -0,0 +1,34 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+#import "RCTBridgeModule.h"
+
+@import FluentUI;
+
+@implementation RCTConvert (MSFButtonAdditions)
+RCT_ENUM_CONVERTER(MSFButtonStyle, (@{
+	@"primary": @(MSFButtonStylePrimaryFilled),
+	@"secondary": @(MSFButtonStyleSecondaryOutline),
+	@"borderless": @(MSFButtonStyleBorderless),
+  @"acrylic": @(MSFButtonStylePrimaryFilled), // mac specific button style
+}), MSFButtonStylePrimaryFilled, integerValue);
+
+@end
+
+@interface RCT_EXTERN_MODULE(MSFUIButtonViewManager, RCTViewManager)
+
+//RCT_EXPORT_VIEW_PROPERTY(image, UIImage);
+RCT_EXPORT_VIEW_PROPERTY(onPress, RCTBubblingEventBlock);
+RCT_REMAP_VIEW_PROPERTY(buttonStyle, style, MSFButtonStyle);
+RCT_CUSTOM_VIEW_PROPERTY(title, String, MSFButton) {
+	// for simplicity only normal state is supported
+	[view setTitle:[RCTConvert NSString:json] forState:normal];
```

Comment:
Setting it for normal sets it for all states, is that fine? https://stackoverflow.com/questions/1375145/is-there-a-way-to-settitle-on-a-uibutton-that-updates-all-uicontrolstates-at-onc

Reply:
I've been looking into extending RCTConvert to add custom types. Anyway, I think it's fine for v1.